kuma/javascript/src/header/__snapshots__/header.test.js.snap

```diff
@@ -405,11 +405,11 @@ exports[`Header snapshot 1`] = `
             role="menuitem"
           >
             <a
-              href="https://bugzilla.mozilla.org/form.mdn"
+              href="https://github.com/mozilla/kuma/issues/new"
               rel="noopener noreferrer"
               target="_blank"
             >
-              Report a bug
+              Report an issue
```

Comment:
For future reference, these `.snap` files are auto generated by [Jest](https://jestjs.io/) so, you do not(should not) update them manually. The way to do it is to run the following from the terminal(https://jestjs.io/docs/en/cli#updatesnapshot):

```
npm t -- --u
```